### DISCUSSION TITLE
تعداد آی پی در هر رنج

### DISCUSSION THREAD

**Discussion by meyroon** [MortezaBashsiz/CFScanner#154] [Ideas]
سلام وقت بخیر
اگر این امکان رو اضافه کنید که کاربر مشخص کنه مثلا از همه رنج آی پی ها فقط 5 تا آی پی که پیدا کرد بره رنج بعدی.
یه فیلد باشه که توش عدد وارد کنیم. برای هر رنج جداگانه نمیگما که خیلی پیچیده بشه. یه فیلد باشه کلا برای همه رنج ها..
اینجوری در حالت فعلی خیلی وقت میگیره خیلی، من میشینم استی Skip میکنم ولی اتوماتیک باشه عالی میشه.
با تشکر از زحمات و وقتتون..

**Comment by goingfine** [comment]
ممنون پیشنهاد خوبی بود اضافه شد

![auto-skip](https://user-images.githubusercontent.com/126115050/223682153-67da51dd-866e-455d-a5ca-078b4e0207e7.png)
د

**Reply by meyroon** [reply]
بسیار مچکرم.. عالی شد.
